services/utils/searchTags.js

```diff
@@ -0,0 +1,21 @@
+// removes leading/trailing whitespaces and extra spaces and adds space after the comma and converted to lowercase
+export const cleanSearchInput = (searchInput) => {
+    return searchInput
+        .trim()
+        .replace(/\s{2,}/g, " ")
+        .replace(/,(?!\s)/g, ", ")
+        .toLowerCase();
+};
+
+export const searchTagNameInInput = (searchInput, tagName) => {
+    const tags = cleanSearchInput(searchInput).split(",");
+
+    for (let tag of tags) {
+        if (tag.trim() === tagName.toLowerCase()) {
+            return true;
+        }
+    }
+
+    return false;
+};
+
```

Comment:
Correct formatting...

```suggestion
export const cleanSearchInput = (searchInput) => {
  return searchInput
    .trim()
    .replace(/\s{2,}/g, " ")
    .replace(/,(?!\s)/g, ", ")
    .toLowerCase();
};

export const searchTagNameInInput = (searchInput, tagName) => {
  const tags = cleanSearchInput(searchInput).split(",");

  for (let tag of tags) {
    if (tag.trim() === tagName.toLowerCase()) {
      return true;
    }
  }

  return false;
};

```